rajawali/src/main/java/c/org/rajawali3d/engine/RenderView.java

```diff
@@ -0,0 +1,66 @@
+package c.org.rajawali3d.engine;
+
+import android.graphics.Rect;
+import android.support.annotation.Nullable;
+import c.org.rajawali3d.annotations.GLThread;
+
+/**
+ * Interface to be implemented by views that are to be managed by the Engine. A RenderView represents
+ * a viewport on the render surface
+ *
+ * @author Jared Woolston (Jared.Woolston@gmail.com)
+ */
+public interface RenderView {
```

Comment:
This is the kind of thing I had in mind when I suggested a layout manager of some sort, but it seems like it could be feature creep and might turn into a can of worms.  Maybe 2.1 or something, until then, leave it up to the developer?